config.json

```diff
@@ -502,9 +502,9 @@
         "spinner-borderColor-light": "#fab9a3",
         "optimizedCheckout-header-backgroundColor": "#555555",
         "optimizedCheckout-headingPrimary-color": "#eeeeee",
-        "optimizedCheckout-headingPrimary-font": "Google_Roboto_400",
+        "optimizedCheckout-headingPrimary-font": "Google_Montserrat_600",
         "optimizedCheckout-headingSecondary-color": "#ffffff",
-        "optimizedCheckout-headingSecondary-font": "Google_Roboto_400",
+        "optimizedCheckout-headingSecondary-font": "Google_Montserrat_600",
```

Comment:
Can we just use 500 or 700 for this? Using this will cause an additional web font to load for very little benefit.

Reply:
oooops, this was actually my mistake, it should have been 700.